bin/sidekiq_web.ru

```diff
@@ -0,0 +1,19 @@
+require "sidekiq"
+require "sidekiq/web"
+
+SESSION_KEY = ".session.key"
+
+if !File.exist?(SESSION_KEY)
+  require "securerandom"
+  File.open(SESSION_KEY, "w") do |f|
+    f.write(SecureRandom.hex(32))
+  end
+end
+
+Sidekiq.configure_client do |config|
+  config.redis = {size: 1}
```

Comment:
I believe this is ensuring that the sidekiq web interface uses only one connection to Redis, which seems fine.